sqlmesh/core/engine_adapter/bigquery.py

```diff
@@ -101,11 +101,18 @@ def create_schema(self, schema_name: str, ignore_if_exists: bool = True) -> None
 
     def columns(self, table_name: TableName) -> t.Dict[str, exp.DataType]:
```

Comment:
Can you add a boolean param to this method called `include_pseudo_columns` (including the base class and all those who override it) and default it to `True`? I wonder if we have some cases where returning pseudo columns is not what we want/expect and if we add this now it will be easy to update those calls to exclude it as needed. It also makes it clear from the function signature that we are returning columns like this if they are defined. The one negative that I see is that other engines may have pseudo columns that we just haven't implemented retrieving them but we can follow up on that later if needed.m

Reply:
Makes sense. Done ✅